### DISCUSSION TITLE
Will there be a release IKVM.Maven.Sdk 1.8.0 this week?

### DISCUSSION THREAD

**Discussion by martin-honnen** [ikvm-revived/ikvm-sdk-maven#68] [Q&A]
First of all, congrats on the IKVM 8.10 release, sounds really great to have an up to date Java 8 JDK in IKVM.

While trying to use it it seems the latest IKVM.Maven.Sdk is a prerelease `<PackageReference Include="IKVM.Maven.Sdk" Version="1.8.0-pre.1" />`, will you do a final release of IKVM.Maven.Sdk during the next days/this week?

I tried to build some sample package with IKVM 8.10 and that prerelease of IKVM.Maven.Sdk and it seems to work out fine but before publishing a package I wonder whether it is better to wait for a final release of IKVM.Maven.Sdk compatible with the latest IKVM.

**Comment by wasabii** [comment]
Yup.
